Classes/Domain/Repository/StoredSearchRepository.php

```diff
@@ -0,0 +1,6 @@
+<?php
+namespace EWW\Dpf\Domain\Repository;
+
+class StoredSearchRepository extends \TYPO3\CMS\Extbase\Persistence\Repository
+{
```

Comment:
In extbase this is regulated by convention, here it would be the domain model StoredSearch. The CRUD methods are all default, thats all we need here.